pynicotine/search.py

```diff
@@ -532,16 +529,14 @@ def process_search_request(self, searchterm, user, searchid, direct=False):
         self.core.send_message_to_peer(user, message)
 
         if direct:
-            log.add_search(
-                _("User %(user)s is directly searching for \"%(query)s\", returning %(num)i results"), {
-                    'user': user,
-                    'query': searchterm_old,
-                    'num': numresults
-                })
+            log.add_search(_("User %(user)s is directly searching for \"%(query)s\", found %(num)i results"), {
```

Comment:
I'd like the log message to mention what's actually happening, e.g. "attempting to send x results".

Reply:
What would be really ideal, is to enter the log in slskproto such that we know whether the results were successfully sent or not (depending upon if the associated connection was established etc), but the logic for tracking the message throughout the course of the connection attempt is probably highly complex.

The pluginsystem has a similar limitation, whereby the incoming search notification is generated too soon to know if there was a hit or a miss, nor how many results were found/sent.